code/__HELPERS/global_lists.dm

```diff
@@ -15,6 +15,12 @@ var/global/list/processing_machines = list()
 var/global/list/active_diseases = list()
 var/global/list/events = list()
 
+//tips
+var/global/list/marinetips = file2list('config/tips/marinetips.txt')
```

Comment:
You could convert all of these into GLOB globalvars.

Reply:
I'll need some examples to work off of.